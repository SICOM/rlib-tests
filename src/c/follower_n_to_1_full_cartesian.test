COMMAND=./follower_n_to_1_full_cartesian
FORMATS="pdf xml txt csv html"
EXPECTED_pdf=
EXPECTED_xml=
EXPECTED_txt=
EXPECTED_csv=
EXPECTED_html=
