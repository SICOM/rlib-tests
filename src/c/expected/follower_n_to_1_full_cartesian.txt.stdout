Test Report (Normal) AND THE REPORT FORMAT IS: TXT       3    
Test Report (Bold)
Test Report (Italics)
Test Report (Bold and Italics)
BOO: mike is a 
First Name           Last Name            Color               
Bob                  Doan                 blue                 WRD   MATCH    MATCH    MATCH    MATCH   
Bob                  Doan                 blue                 TD    NO_MATCH NO_MATCH NO_MATCH NO_MATCH
Bob                  Doan                 blue                 ERB   NO_MATCH NO_MATCH NO_MATCH NO_MATCH
Terry                Doan                 blue                 WRD   NO_MATCH NO_MATCH NO_MATCH NO_MATCH
Terry                Doan                 blue                 TD    MATCH    MATCH    MATCH    MATCH   
Terry                Doan                 blue                 ERB   NO_MATCH NO_MATCH NO_MATCH NO_MATCH
Eric                 Buruschkin           green                WRD   NO_MATCH NO_MATCH NO_MATCH NO_MATCH
Eric                 Buruschkin           green                TD    NO_MATCH NO_MATCH NO_MATCH NO_MATCH
Eric                 Buruschkin           green                ERB   MATCH    MATCH    MATCH    MATCH   
Mike                 Roth                 yellow               WRD   NO_MATCH NO_MATCH NO_MATCH NO_MATCH
Mike                 Roth                 yellow               TD    NO_MATCH NO_MATCH NO_MATCH NO_MATCH
Mike                 Roth                 yellow               ERB   NO_MATCH NO_MATCH NO_MATCH NO_MATCH
Bob                  Kratz                pink                 WRD   NO_MATCH NO_MATCH NO_MATCH NO_MATCH
Bob                  Kratz                pink                 TD    NO_MATCH NO_MATCH NO_MATCH NO_MATCH
Bob                  Kratz                pink                 ERB   NO_MATCH NO_MATCH NO_MATCH NO_MATCH
Steve                Tilden               purple               WRD   NO_MATCH NO_MATCH NO_MATCH NO_MATCH
Steve                Tilden               purple               TD    NO_MATCH NO_MATCH NO_MATCH NO_MATCH
Steve                Tilden               purple               ERB   NO_MATCH NO_MATCH NO_MATCH NO_MATCH
Page:   1
